test/scrivener/paginator/ecto/query_test.exs

```diff
@@ -141,7 +141,18 @@ defmodule Scrivener.Paginator.Ecto.QueryTest do
         |> group_by([p], p.id)
         |> Scrivener.Ecto.Repo.paginate
 
-      assert page.total_entries == 7
+      assert page.total_entries == 8
```

Comment:
This change to `8` is exactly why I did follow this path to begin with. We want the `total_entries` value to respect the grouping.